<a href="https://colab.research.google.com/github/BDBandito/foreveryone1/blob/main/A1_1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

**1.1**

The liquid drop model determines the binding energy of a nucleus to be:

$$
BE(Z, A) = a_V A - a_S A^{2/3} - a_C \frac{Z^2}{A^{1/3}} - a_A \frac{(Z - A/2)^2}{A} + \frac{a_P}{A^{1/2}} \delta(Z, A)
$$

$a_v, a_s, a_C, a_A$ and $a_P$ are all constants and given by table 1. A is the mass number and Z is the atom number of a specified nucleus. $δ(A,Z)$ is determined by:

$$
\begin{aligned}
\delta(A, Z) =
\begin{cases}
1, & \text{if } Z \text{ even, } N \text{ even } (A \text{ even}) \\
0, & \text{if } A \text{ odd} \\
-1, & \text{if } Z \text{ odd, } N \text{ odd } (A \text{ even})
\end{cases}
\end{aligned}
$$

Below, the binding energies of ${}^{237}\text{Np}$ and ${}^{238}\text{Np}$ are calculated using the liquid drop model.

In [7]:
import numpy as np
import pandas as pd

def pairing_delta(Z, A):
    if A % 2 == 1:  # Odd mass number
        return 0
    elif Z % 2 == 0:  # Even-even nucleus
        return 1
    else:  # Odd-odd nucleus
        return -1

# Constants from table 1 [GeV]
a_V = 0.01567
a_S = 0.01723
a_C = 0.000714
a_A = 0.02315
a_P = 0.0112

# Neptunium properties
Z = 93  # Atomic number of Np
A_values = [237, 238]  # Mass numbers for Np-237 and Np-238

def binding_energy(A, Z):
    strong_force = a_V * A
    surface_tension = a_S * A**(2/3)
    coulomb_repulsion = a_C * (Z**2) / A**(1/3)
    asymmetry = a_A * ((Z - A / 2)**2) / A
    spin_coupling = (a_P / A**0.5) * pairing_delta(Z, A)

    B = strong_force - surface_tension - coulomb_repulsion - asymmetry + spin_coupling
    return B

# Compute  binding energies for Np-237 and Np-238
binding_energies = {A: binding_energy(A, Z) for A in A_values}

# Printing results
df = pd.DataFrame({
    "Isotope": [f"{Z}Np-{A}" for A in A_values],
    "Binding Energy (GeV)": [binding_energies[A] for A in A_values]
})

print(df)




    Isotope  Binding Energy (GeV)
0  93Np-237              1.992529
1  93Np-238              2.004780


**1.2**

The nuclear data presented in the table below was retrieved from the KAERI Nuclear Data Center {https://atom.kaeri.re.kr/nuchart/?zlv=1#}.

\begin{array}{|c|c|c|}
\hline
\text{Isotope} & \text{Binding Energy KAERI (GeV)} & \text{Binding Energy Calculated (GeV)}\\
\hline
^{237}\text{Np} & 1.79525 & 1.99253\\
^{238}\text{Np} & 1.80076 & 2.00478\\
\hline
\end{array}

There is a difference of about $0.2\ GeV$ between the tabulated and calculated values.

**1.3**

The exhitation energy is given by:

$$
E_{exc} = BE({}^{237}Np)+ E_{neutron}+ KE_{neutron} - BE({}^{238}Np)
$$

$KE_{neutron}$ is assumed to be $\approx0$

In [12]:
E_neutron = 0.93957 # GeV

E_exc = binding_energies[237] + E_n - binding_energies[238]
print("Excitation energy in GeV")
print(np.abs(E_exc))

energy in GeV
0.012250934751214748


**1.4**

Calculated excitation energy is $12.5\ MeV$ and the neutron separation energy grabbed from the internet for ${}^{238}Np$ is about $5.4\ MeV$. The difference is then $7.1\ MeV$.

**1.5**

The critical energy is given by:

$$
E_c = A^{\frac{2}{3}} \cdot \left( 0.884 - 0.01989 \cdot \frac{Z^2}{A} \right)
$$

The critical energy for Np37 is provided below.


In [14]:
def E_c(A, Z):
    return (A**(2/3)) * (0.884 - 0.01989 * (Z**2 / A))

A = 237  # Mass number Np37
Z = 93   # Atomic numbe Np37

E_c_value = E_c(A, Z)
print(f"E_c = {E_c_value} MeV")


E_c = 6.056332798722767 MeV


**1.6**

Because $E_{exc}$ is higher than $E_c$, the isotope is fissile.  